### PR TITLE
Got blinky to work

### PR DESCRIPTION
**Pull request by dhylands**
I think that we need to cleanup a bunch of stuff, but it works.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+TARGET=thumbv7em-none-eabi
+ELF_DIR=target/${TARGET}/debug
+FIRMWARE_ELF=${ELF_DIR}/firmware
+FIRMWARE_BIN=${ELF_DIR}/firmware.bin
+
+rm -f ${FIRMWARE_ELF}
+rm -f ${FIRMWARE_BIN}
+
+#xargo build --target=cortex-m4 "$@"
```
Comment (fabricedesre) [inline build.sh]: just remove that line

File: cc3200-sys/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cc3200-sys"
+version = "0.1.0"
+authors = ["Dave Hylands <dhylands@gmail.com>"]
+build = "build.rs"
+
+#[dependencies]
+#libc = "0.2"
```
Comment (fabricedesre) [inline cc3200-sys/Cargo.toml]: that too

File: cc3200-sys/board.c [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    // when it loads the program into RAM.
+
+    //
+    // Zero fill the bss segment.
+    //
+    __asm("    ldr     r0, =_bss\n"
+          "    ldr     r1, =_ebss\n"
+          "    mov     r2, #0\n"
+          "    .thumb_func\n"
+          "zero_loop:\n"
```
Comment (fabricedesre) [inline cc3200-sys/board.c:37]: indentation issue?

**Review by fabricedesre — APPROVED** [review]
Can you also remove the commented lines at the beginning of cc3200-sys/src/lib.rs ?

File: cc3200-sys/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,48 @@
+//extern crate libc;
+
+//use libc::{ c_void };
+
```
Comment (fabricedesre) [inline cc3200-sys/src/lib.rs]: remove these commented lines

File: cc3200-sys/board.c (hunk 3 above)
Comment (dhylands) [reply cc3200-sys/board.c:37]: Heh - copy/paste

**Review by fabricedesre — APPROVED** [review]

**Pull request merged**